modules/doc/content/newsletter/2024/2024_12.md

```diff
@@ -1,18 +1,63 @@
 # MOOSE Newsletter (December 2024)
 
-!alert! construction title=In Progress
-This MOOSE Newsletter edition is in progress. Please check back in January 2025
-for a complete description of all MOOSE changes.
+!alert! note title=INL Holiday Curtailment
+[!ac](INL) was closed for its annual Curtailment of Operations December 25, 2024 through January 1, 2025.
+Accordingly, development of MOOSE and developer/user support was very light during this period. The
+MOOSE development team hopes that all of our collaborators, users, and colleagues around the world
+had a safe and restful holiday season!
 !alert-end!
 
 ## MOOSE Improvements
 
+### Transient capabilities for the linear finite volume discretization
+
+The linear finite volume discretization is used in the Navier Stokes module notably to set up segregated solves
+for pressure-velocity coupling. This discretization was first implemented without support for time integration.
+This has now been implemented, with new APIs which should be implemented in each of the `TimeIntegrator` classes.
+These routines are then leveraged by the new [LinearFVTimeDerivative.md] kernel during the RHS and matrix assembly.
+
+- `timeDerivativeRHSContribution(..)` for the contribution of a time derivative to the right hand side of a linear system
+- `timeDerivativeMatrixContribution(..)` for the contribution of a time derivative to the matrix of a linear system
+
+Currently, these routines have only been implemented for the first order [ImplicitEuler.md] time integrator.
```

Comment:
@grmnptr

Reply:
Nice, thanks @GiudGiud for adding these to the newsletter!